maintainers/scripts/pluginupdate.py

```diff
@@ -797,7 +900,12 @@ def update_plugins(editor: Editor, args):
         "Either set --proc=1 or --github-token=YOUR_TOKEN. ")
 
     fetch_config = FetchConfig(args.proc, args.github_token)
-    update = editor.get_update(args.input_file, args.outfile, fetch_config)
+    update = editor.get_update(
+        args.input_file,
+        args.outfile,
+        fetch_config,
+        getattr(args, "update_only", None),  # if script was called without arguments
```

Comment:
I recommend using names for the parameters here.
```suggestion
        input_file=args.input_file,
        output_file=args.outfile,
        config=fetch_config,
        to_update=getattr(args, "update_only", None),  # if script was called without arguments
```